### PR TITLE
Add initial Renovate configuration

### PR DESCRIPTION
**Pull request by mallardduck**
#### Types of Change ####
This should allow renovate bot to send PRs that will help ensure that GHAs are kept updated.
It will help keep things updated on-going and address the existing GHA on old versions.

This will help address this specific warning we see on workflows:

> The following actions uses Node.js version which is deprecated and will be forced to run on node20: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/

### CODE REVIEW

File: .github/workflows/renovate.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  schedule:
+    - cron: '30 4,6 * * *'
+
+jobs:
+  call-workflow:
+    uses: rancher/renovate-config/.github/workflows/renovate.yml@release
+    with:
+      logLevel: ${{ inputs.logLevel || 'info' }}
+      overrideSchedule: ${{ github.event.inputs.overrideSchedule == 'true' && '{''schedule'':null}' || '' }}
+    secrets: inherit
```
Comment (pjbgf) [inline .github/workflows/renovate.yml:25]:
```suggestion
    secrets: inherit
    
```

File: .github/renovate.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["github>rancher/renovate-config#release"],
+  "baseBranches": ["master"]
+}
```
Comment (pjbgf) [inline .github/renovate.json:4]:
```suggestion
}

```

**Review by pjbgf — COMMENTED** [review]
Two nits, otherwise LGTM.

**Comment by mallardduck** [comment]
Thank you @pjbgf - updated to correct those.

**Review by pjbgf — APPROVED** [review]

**Review by brandond — COMMENTED** [review]
Just to be clear - we're just using renovate to update the GHA workflows, not to touch anything related to image mirroring itself. LGTM!

**Review by brandond — APPROVED** [review]

**Pull request merged**